ci/vars-logging-test.yml

```
@@ -0,0 +1,7 @@
+---
+# temp: skip os-net-setup
+cifmw_os_net_setup_config: []
+post_deploy:
+  - name: run logging functional test
+    source: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}/ci/run_playbooks_logging.yml"
+    type: playbook
```

The hook should be defined using the the "single parameter" format.

Further information is here:
https://github.com/infrawatch/feature-verification-tests/commit/a63b85103015a168656937fb51c40461c59f24ec

```suggestion
post_deploy_run_logging_functional_test:
  source: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}/ci/run_playbooks_logging.yml"
  type: playbook
```